### PR TITLE
Add Decode Morse Methods

### PR DESCRIPTION
**Pull request by AbdusSattar-70**
### In this milestone, we have followed a pair-programming approach to Create methods to decode the Morse codes.

### CODE REVIEW

File: morsecode.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  @morse_code_to_character[morse_code]
+end
+
+def decode_word(words)
+  @words_array = words.split
+  @decoded_word = ''
+  @words_array.each do |word|
+    @decoded_word += decode_char(word)
+  end
+  print "#{@decoded_word} "
```
Comment (TedLivist) [inline morsecode.rb]: - [x] We should return the decoded word, not `print` it. Kindly fix that.

- [x] Also remove the whitespace at the end of the returned word

  ```
  "#{@decoded_word}"
  ```

File: morsecode.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+# decode_word("-- -.--")
+
+def decode(message)
+  @message_array = message.split('   ')
+  @message_array.each do |messages|
+    @decoded_messages = decode_word(messages)
+  end
+  @decoded_messages
+end
```
Comment (TedLivist) [inline morsecode.rb]: - [x] After resolving the issue above this method no longer works as expected. Kindly resolve that. Here is a suggestion:

  ```
  def decode(message)
    @message_array = message.split('   ')
    @decoded_messages = ''
    @message_array.each do |messages|
      @decoded_messages += "#{decode_word(messages)} "
    end
    @decoded_messages.strip
  end
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @AbdusSattar-70 and @LoviMadaan,

Good job so far!
✔️ Reusable methods
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbdusSattar-70** [comment]
@TedLivist Thanks for your valuable suggestions. We will fix it.

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @AbdusSattar-70 & @LoviMadaan 🙋‍♀️

Great work implementing the changes suggested by the previous reviewer 💪**_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Decoded well, perfect!
-  ✅ Great work using the right Gitflow 
-  ✅ Your README looks professional, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbdusSattar-70** [comment]
@KanzaTahreem Thanks for the approval.

**Pull request merged**